src/screens/CreatePost/elements/SheetAddTopic.js

```diff
@@ -250,7 +254,7 @@ const styles = StyleSheet.create({
   textDesc: {
     fontSize: 10,
     fontFamily: fonts.inter[400],
-    color: colors.gray,
+    color: COLORS.gray8,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color value has been updated to use a constant from the `COLORS` object. This change is consistent with the PR summary and improves maintainability by centralizing color definitions.
```diff
- color: colors.gray,
+ color: COLORS.gray8,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->